main.go

```diff
@@ -10,33 +10,33 @@ import (
 	alpm "github.com/jguer/go-alpm"
 )
 
-func setPaths() error {
+func setPaths() (err error) {
```

Comment:
There's no need to define err here, we don't use it nor do we use a naked return. Apart from that nitpick its good.

Reply:
Fixed :smile: